src/PlatformEngines.jl

```diff
@@ -451,7 +451,9 @@ function download(url::AbstractString, dest::AbstractString;
         if isa(e, InterruptException)
             rethrow()
         end
-        error("Could not download $(url) to $(dest)")
+        st = stacktrace(catch_backtrace())
+        @error "Could not download $(url) to $(dest)" exception=(e, st)
```

Comment:
Why is this necessary?

Reply:
Not strictly necessary, but it enabled me to actually see what error was being thrown instead of swallowing it in a generic `error` call.